README.md

```diff
@@ -183,6 +184,38 @@ Once in the web app:
 * Explore citations and sources
 * Click on "settings" to try different options, tweak prompts, etc.
 
+## Productionizing
+
+This sample is designed to be a starting point for your own production application,
+but you should do a thorough review of the security and performance before deploying
+to production. Here are some things to consider:
+
+* **OpenAI Capacity**: The default TPM (tokens per minute) is set to 30K. That is equivalent
+  to approximately 30 conversations per minute (assuming 1K per user message/response).
+  You can increase the capacity by changing the `chatGptDeploymentCapacity` and `embeddingDeploymentCapacity` parameters in `infra/main.bicep` to your account's maximum capacity.
+  You can also view the Quotas tab from Azure OpenAI studio to understand how much capacity you have.
+* **Azure Storage**: The default storage account uses the `Standard_LRS` SKU.
+  We recommend using `Standard_ZRS` for production deployments,
+  which you can specify using the `sku` property in `infra/main.bicep`.
+* **Azure Cognitive Search**: The default search service uses the `Standard` SKU
+  with the free semantic search option. You should either change `semanticSearch` to "standard"
+  or disable semantic search entirely in the approaches files. If you see errors about search service capacity being exceeded, you may find it helpful to increase the number of replicas by changing `replicaCount` in `infra/core/search/search-services.bicep`.
```

Comment:
in the /app/backend/approaches files.

Reply:
I think it's in `infra/core/search/search-services.bicep`, or manually scaling it from the Azure Portal.